doc/releases/quincy.rst

```diff
@@ -5,6 +5,44 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.5 Quincy
+==============
+This is the fifth hotfix release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* Treating daemons as `<daemon_type>.x` caused a crash
+in the Telemetry module since the current method does not cover a case
+where a daemon id is more complex, i.e. `<daemon_type>.x.y`.
+When we parse the daemon type and daemon id, we should
+split it into a maximum of two pieces rather than splitting
+it by every `.` character. Specifying `1` in the Python
+.split() function will limit the split to a maximum of two items.
+Fixes: https://tracker.ceph.com/issues/57700
+Signed-off-by: Laura Flores <lflores@redhat.com>
+
+*  bea9f4b introduced a regression that makes the activate process
+take a very long time to complete.
+`_get_bluestore_info()` which calls `ceph-bluestore-tool` binary via
+subprocess is called in an exponential way while this is not needed.
+Fixes: https://tracker.ceph.com/issues/57627
+Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>
+
+* Resolves: https://tracker.ceph.com/issues/55605
```

Comment:
don't need to include this here, the short description in the changelog is good enough